packages/core/forms/src/generator/fields/advanced/FieldArrayCardContainer.vue

```diff
@@ -51,12 +44,16 @@ export default {
   align-items: center;
   display: flex;
 
+  :deep(.kong-card) {
```

Comment:
`.kong-card` is the old class name for `KCard` before reskinning, so this is not hitting any element since `KCard` reskin. I think it's OK to just remove this block?